src/components/node-list/styles/_row.scss

```diff
@@ -114,39 +95,16 @@
   white-space: nowrap;
   text-overflow: ellipsis;
 
-  .kui-theme--light & {
-    color: rgba($color-dark, 0.8);
-  }
-
-  .kui-theme--dark & {
-    color: rgba($color-light, 0.8);
-  }
-
-  .kui-theme--dark &--faded {
-    color: rgba($color-light, 0.55);
-  }
-
-  .kui-theme--light &--faded {
-    color: rgba($color-dark, 0.55);
-  }
-
-  .kui-theme--dark &--disabled {
-    color: rgba($color-light, 0.25);
-  }
-
-  .kui-theme--light &--disabled {
-    color: rgba($color-dark, 0.25);
-  }
-
-  .kui-theme--dark & b {
-    color: $color-primary;
+  &--faded {
+    opacity: 0.65;
   }
 
-  .kui-theme--light & b {
-    color: darken($color-primary, 10%);
+  &--disabled {
+    opacity: 0.3;
```

Comment:
same as the above - is the `color` setting replaced by opacity?

Reply:
yup, 0.25 / 0.85 = 0.3